docs/developers/developing_iree/sanitizers.md

```diff
@@ -23,39 +23,66 @@ Notes:
   documentation](https://clang.llvm.org/docs/AddressSanitizer.html#memory-leak-detection)
   on leak detection. It is only enabled by default on some platforms.
 
-## Enabling the sanitizers
+## Support status and how to enable each sanitizer
 
-In the CMake build system of IREE, at least on Linux and Android, enabling these
-sanitizers is a simple matter of passing one of these options to the initial
-CMake command:
+### ASan (AddressSanitizer)
+
+Enabling ASan in the IREE build is a simple matter of setting the
+`IREE_ENABLE_ASAN` CMake option:
 
 ```
--DIREE_ENABLE_ASAN=ON
+cmake -DIREE_ENABLE_ASAN=ON ...
 ```
 
-or
+### TSan (ThreadSanitizer)
+
+Enabling TSan in the IREE build is, in itself, a simple matter of setting the
+`IREE_ENABLE_TSAN` CMake option:
 
 ```
--DIREE_ENABLE_MSAN=ON
+cmake -DIREE_ENABLE_TSAN=ON ...
 ```
 
-or
+That enables TSan in all the C/C++ targets, which includes the IREE runtime,
+the IREE compiler, and all the C/C++ unit tests.
+
+The resuting TSan-instrumented IREE runtime is only able to load IREE modules
+that have also been built with TSan instrumentation, as enabled by passing
+`iree-llvm-sanitize=thread` to the IREE compiler. That in turn also requires
+passing `-iree-llvm-link-embedded=false` to the IREE compiler.
```

Comment:
This only applies when using the CPU codegen compiler target, right? (VMVX, Vulkan, CUDA, etc. executables won't benefit from TSan instrumentation but also won't be incompatible)

Reply:
Put another way, I'd expect IREE modules (.vmfb files) produced by those compiler targets to be byte-for-byte identical regardless of sanitizer setting. The is no CPU code in them for the sanitizers to affect.

Reply:
I'd just qualify with "CPU" or "LLVM" when talking about "IREE modules", like:

"When using the LLVM/CPU compiler target with a TSan-instrumented runtime, TSan instrumentation must be included in generated modules by passing the `--iree-llvm-sanitize=thread` and `--iree-llvm-link-embedded=false` flags to the IREE compiler."

Reply:
OK, edited. Different phrasing to phrase more directly what the root issue is but now mentioning LLVM/CPU.